jetty-core/jetty-util/src/main/java/org/eclipse/jetty/util/BlockUtil.java

```diff
@@ -31,7 +31,7 @@
  *
  * <h2>Non shared Runnable</h2>
  * <pre>
- *     try(Blocking.Runnable onAction = Blocking.runnable())
+ *     try(BlockUtil.Runnable onAction = BlockUtil.runnable())
```

Comment:
I liked `Blocking` much more than `BlockUtil`.

Reply:
So then what should I call `Handler.Blocking`?   I renamed to avoid a name clash.
Happy to rename it back, but need a better name for the handler.